docs/casa/manual/chapters/10_outer_frame_assembly.md

```diff
@@ -0,0 +1,151 @@
+# Outer Frame Assembly
+
+![](../img/10_1.png)
+
+The outer frame assembly of Casa, designed using 2020 V slot extrusion, is assembled in two halves. Each half is joined by a backbone of 2040 extrusion, making the enclosure easy to disassemble for moving. The panels, sliding door, electronics, and basement will attach to this main frame. Throughout this stage, remember to check for squareness and adjust accordingly.
+
+The outer frame assembly uses eight different extrusions labeled A - H. Some extrusions are the same length but feature different cross-drill patterns, so pay attention and select the correct extrusion. Extrusions A, C, D, and H are the extrusions that have cross-drilling. If you are self-sourcing, you may find the printable cross-drill guides handy.
+
+<!-- We really need a way of defining the extrusions here -->
```

Comment:
We should also find some way to communicate that the frame can be assembled with the electronics on the left or right to suit the users' particular needs.

Reply:
Added a tip block to mention inverting the frame and/or door